src/Parse/Date.php

```diff
@@ -726,7 +728,7 @@ public function date_w3cdtf($date)
             // Convert the number of seconds to an integer, taking decimals into account
             $second = round((int)$match[6] + (int)$match[7] / (10 ** strlen($match[7])));
 
-            return gmmktime($match[4], $match[5], $second, $match[2], $match[3], $match[1]) - $timezone;
+            return gmmktime(intval($match[4]), intval($match[5]), intval($second), intval($match[2]), intval($match[3]), intval($match[1])) - $timezone;
```

Comment:
Ideally, we would convert it into a proper type as soon as possible (e.g. https://github.com/simplepie/simplepie/pull/765) but this looks good for now.